tools/ai_sidecar/coaching/llm_coach.py

```
@@ -306,3 +311,124 @@ def compose_debrief(
         return llm
     logger.info("ollama unreachable or empty; using rules debrief")
     return rules
+
+
+def compose_corner_hint(
+    corner: str,
+    cur_kmh: float,
+    ref_kmh: float,
+    dist_m: float,
+    car: str | None = None,
```

Ruff (0.15.9)

[warning] Unused function argument: `car`

(ARG001)